profiles/models.py

```diff
@@ -68,5 +75,22 @@ def is_complete(self):
                 return False
         return True
 
+    @transaction.atomic
+    def save(self, *args, update_image=False, **kwargs):  # pylint: disable=arguments-differ
+        """Update thumbnails if necessary"""
+        if update_image:
+            if self.image_file:
+                small_thumbnail = make_thumbnail(self.image_file.file, IMAGE_SMALL_MAX_DIMENSION)
```

Comment:
Is there a meaningful difference between `self.image_file.file` here and `self.image_file` below?

Reply:
First is an `ImageFieldFile` object, the second is a `File` object.  I think the `make_thumbnail` function only works with the latter.
```python
In [3]: profile.image_file
Out[3]: <ImageFieldFile: profile/01CFDJ0PE2S2PQ9QFFQD7MGMR7/0-2018-06-08T183816.jpg>

In [4]: profile.image_file.file
Out[4]: <File: /var/media/profile/01CFDJ0PE2S2PQ9QFFQD7MGMR7/0-2018-06-08T183816.jpg>
```

Reply:
Actually nvm, seems like either will work, will change